apps/monorail/views/Skus.js

```diff
@@ -2,26 +2,12 @@
 
 'use strict';
 
-/* eslint-disable no-unused-vars */
 import React, { Component } from 'react';
-import mixin from 'common-web-ui/lib/mixin';
-import PageHelpers from 'common-web-ui/mixins/PageHelpers';
-/* eslint-enable no-unused-vars */
 
-import {} from 'material-ui';
 import SkusGrid from './SkusGrid';
 
-@mixin(PageHelpers)
 export default class Skus extends Component {
 
-  render() {
-    return (
-      <div className="Skus">
-        {this.renderBreadcrumbs({href: '', label: 'Dashboard'}, 'Skus')}
-        <SkusGrid />
-      </div>
-    );
-
-  }
+  render() { return <SkusGrid />; }
```

Comment:
Expected an identifier and instead saw '<'.<br>Missing semicolon.<br>Expected an identifier and instead saw '>'.<br>Expected an assignment or function call and instead saw an expression.